README.md

```diff
@@ -1,12 +1,10 @@
-<h1 style="color: steelblue;">⚠️ Upcoming changes ⚠️</h1>
+<h1 style="color: steelblue;">⚠️ New standard for Bicep modules - AVM ⚠️</h1>
 
-To provide our customers with a unified experience, Azure Verified Modules ([AVM](https://aka.ms/AVM)) will become the single Microsoft standard for Bicep modules, published to the Public Bicep Registry, via this repository.
+To provide our customers with a unified experience, Azure Verified Modules ([AVM](https://aka.ms/AVM)) is now the single Microsoft standard for Bicep modules, published to the Public Bicep Registry, via this repository.
 
-Going forward, new modules will need to be developed and published in accordance with the [AVM specifications](https://azure.github.io/Azure-Verified-Modules/specs/module-specs/). Module proposals for new, non-AVM modules will no longer be accepted. To propose a new AVM module, you can file a Module Proposal, [here](https://aka.ms/AVM/ModuleProposal).
+Going forward, new modules need to be developed and published in accordance with the [AVM specifications](https://azure.github.io/Azure-Verified-Modules/specs/module-specs/). Module proposals for new, non-AVM modules are no longer accepted. To propose a new AVM module, you can file a Module Proposal, [here](https://aka.ms/AVM/ModuleProposal).
```

Comment:
We could add [GH markdown alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) to highlight important info, as recently done for automatic replies. For example

```suggestion
Going forward, new modules need to be developed and published in accordance with the [AVM specifications](https://azure.github.io/Azure-Verified-Modules/specs/module-specs/). 
> [!CAUTION]
> Module proposals for new, non-AVM modules are no longer accepted. 
To propose a new AVM module, you can file a Module Proposal, [here](https://aka.ms/AVM/ModuleProposal).
```

Reply:
I didn't want to make the landing page overly colorful 😊, but as long as it's only one callout, I think we're good. Following the official definitions for the various levels of call-outs however, I'd go with `IMPORTANT` and not `CAUTION`.
<img width="373" alt="image" src="https://github.com/Azure/bicep-registry-modules/assets/22733424/6d665e76-b8fa-4c37-89f3-57b4549e11f5">

Reply:
Sounds fair, it was only an example, not an actual suggestion. You can resolve the comment.